update Observability tech references (#12098)
kyma-project/kyma: docs/05-technical-reference/00-configuration-parameters/README.md

title: Configuration parameters
---

Kyma uses Helm's chart templating mechanism with `values.yaml` files that contain the configuration parameters.

> A global chart can override the subcharts that complement it. Learn more about the syntax and semantic of [global charts and subcharts in Helm](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/).

Reviewer: ```suggestion
A global chart can override the subcharts that complement it. Learn more about the syntax and semantics of [global charts and subcharts in Helm](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/).
```

> If you want to [change the Kyma settings](../../04-operation-guides/operations/03-change-kyma-config-values.md), you simply modify the values of the respective chart or subchart, and deploy them.

Reviewer 2: ```suggestion
If you want to [change the Kyma settings](../../04-operation-guides/operations/03-change-kyma-config-values.md), you simply re-define them in your custom values.yaml file and supply them as input to the "kyma deploy" command, similar to helm commands.
```

Author: thanks!

> If you want to [change the Kyma settings](../../04-operation-guides/operations/03-change-kyma-config-values.md), you simply redefine them in your custom `values.yaml` file and supply them as input to the `kyma deploy` command, similar to Helm commands.

Reviewer: ```suggestion
If you want to [change the Kyma settings](../../04-operation-guides/operations/03-change-kyma-config-values.md), simply redefine them in your custom `values.yaml` file and supply them as input to the `kyma deploy` command, similar to Helm commands.
```
Removed `you`.
Nitpicky again, I'll leave it up to you.

Author: I love nitpicky review! Thank you :)
